.env.example

```
@@ -0,0 +1 @@
+VITE_TASK_PER_PAGE=9
```

Avoid EOF please. Also use prettier.
```suggestion
VITE_TASK_PER_PAGE = 9
```